src/image/ImageAnnotator.ts

```diff
@@ -0,0 +1,87 @@
+import BaseAnnotator from '../common/BaseAnnotator';
+import BaseManager from '../common/BaseManager';
+import { CLASS_ANNOTATIONS_LOADED } from '../constants';
+import { getAnnotation } from '../store/annotations';
+import { centerRegion, isRegion, RegionManager } from '../region';
+import './ImageAnnotator.scss';
+import { scrollToLocation } from '../utils/scroll';
+
+export default class ImageAnnotator extends BaseAnnotator {
+    annotatedEl?: HTMLElement;
+
+    managers: Set<BaseManager> = new Set();
+
+    getManagers(parentEl: HTMLElement, referenceEl: HTMLElement): Set<BaseManager> {
+        this.managers.forEach(manager => {
+            if (!manager.exists(parentEl)) {
+                manager.destroy();
+                this.managers.delete(manager);
+            }
+        });
+
+        if (this.managers.size === 0) {
+            this.managers.add(new RegionManager({ referenceEl }));
+        }
+
+        return this.managers;
+    }
+
+    getReference(): HTMLElement | null {
+        return this.annotatedEl ? this.annotatedEl.querySelector('img') : null;
+    }
+
+    init(scale = 1, rotation = 0): void {
+        super.init(scale, rotation);
+
+        if (!this.rootEl) {
+            return;
+        }
+
+        this.annotatedEl = this.rootEl.querySelector('.bp-image') as HTMLElement;
+        this.annotatedEl.classList.add(CLASS_ANNOTATIONS_LOADED);
+
+        this.render();
+    }
+
+    render(): void {
+        const referenceEl = this.getReference();
+
+        if (!this.annotatedEl || !referenceEl) {
+            return;
+        }
+
+        this.getManagers(this.annotatedEl, referenceEl).forEach(manager => {
+            manager.style({
```

Comment:
This feels a little sketchy to me and can probably be refined. Unfortunately, we're not able to rely on the position/size of a parent element for images, as the `img` tag itself is sized directly by the Preview SDK. This solution keeps the annotation rendering area (ba-Layer) in sync with the image element.